packages/appium/docs/en/guides/caps.md

```diff
@@ -2,9 +2,150 @@
 title: Capabilities
 ---
 
-TODO
+"Capabilities" is the name given to the set of parameters used to start an Appium session. The
+information in the set is used to describe what sort of "capabilities" you want your session to
+have, for example, a certain mobile operating system or a certain version of a device. When you
+start your Appium session, your Appium client will include the set of capabilities you've defined,
+as an object in the JSON-formatted body of the request. Capabilities are represented as key-value
+pairs, with values allowed to be any valid JSON type, including other objects. Appium will then
+examine the capabilities and make sure that it can satisfy them before proceeding to start the
+session and return an ID representing the session to your client library.
 
-## Special Notes for Cloud Providers
+The W3C WebDriver spec's [section on Capabilities](https://w3c.github.io/webdriver/#capabilities)
+identifies a small set of 10 standard capabilities, including the following:
+
+| Capability Name  | Type     | Description                                    |
+|------------------|----------|------------------------------------------------|
+| `browserName`    | `string` | The name of the browser to launch and automate |
+| `browserVersion` | `string` | The specific version of the browser            |
+| `platformName`   | `string` | The type of platform hosting the browser       |
+
+## Appium capabilities
+
+Appium understands these browser-focused capabilities, but introduces a number of additional
+capabilities you are free to use to start Appium sessions. According to the WebDriver spec, any
+non-standard "extension capabilities" must include a namespace prefix (signifying the vendor
+introducing the capability), separated from the capability name by a `:`. Appium's vendor prefix is
+`appium:`, and so any Appium-specific capabilities must include this prefix. Depending on which
+client you are using, the prefix may be added automatically, or you might need to ensure they are
+included in capability names yourself (prefixes are added, for example, in the Appium Python
```

Comment:
actually, this depends on how these capabilities are passed. If the Options concept is used then the above is true and the prefix is added automatically unless some other prefix is already specific. Hoverer if one passes capabilities directly (as dictionary) then no magic happens

Reply:
I wonder if it would be best to just recommend people use the spec.

Reply:
yeah, will just recommend people use it for clarity